test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
check if this is necessary or add the test

Reply:
What kind of test can I add if I'm just including a view and the styles for it?